sources/web/datalab/static/datalab.js

```diff
@@ -47,13 +47,16 @@ function xhr(url, callback, method) {
   request.send();
 }
 
-function initializeDataLab(ipy, events, dialog, utils, security) {
+function initializeDataLab(
+    ipy, events, dialog, utils, security, appbar, editapp,
```

Comment:
I removed the unused args. I moved minibartool to the end with the other unused args to keep the matchup between the module names on lines 108 and 111.